data/hai/hai missions.txt

```diff
@@ -1438,14 +1438,26 @@ mission "Strider: Coalition 2"
 				accept
 
 
-
+# For compatibility
 event "pond strider mass production"
-	shipyard "Hai Basics"
-		"Flea"
-	shipyard "Hai Advanced"
-		"Pond Strider"
-		"Flea"
-	outfitter "Hai Basic"
+
+# The spike in interest leads to ladybugs appearing in the Hai Merchant (Human) fleets.	
```

Comment:
This mission/event is about the Pond Strider, but the comment says Ladybugs. Copy/pasted?